recipes/chaiscript/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(test_package CXX)
+cmake_minimum_required(VERSION 2.8.12)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(chaiscript REQUIRED CONFIG)
+find_package(Threads)
+
+# TEST_PACKAGE #################################################################
+
+if(NOT WIN32)
+    link_libraries(dl)
+endif()
+
+add_executable(${CMAKE_PROJECT_NAME} test_package.cpp)
+target_link_libraries(${CMAKE_PROJECT_NAME} CONAN_PKG::chaiscript Threads::Threads)
+set_property(TARGET ${CMAKE_PROJECT_NAME} PROPERTY CXX_STANDARD 17)
```

Comment:
chaiscript requires a C++14 compiler, so I think 14 suffices.

Reply:
Oh yes, sorry I was looking at the develop branch